integration-cli/docker_cli_nat_test.go

```diff
@@ -75,7 +76,13 @@ func (s *DockerSuite) TestNetworkNat(c *check.C) {
 	}
 
 	result := getContainerLogs(c, srv)
-	if expected := "hello world"; result != expected {
+
+	// Ideally we'd like to check for "hello world" but sometimes
+	// nc doesn't show the data it received so instead let's look for
+	// the output of the 'echo bye' that should be printed once
+	// the nc command gets a connection
+	expected := "bye"
+	if !strings.Contains(result, expected) {
```

Comment:
@duglin fair point. but since we are anyways running these tests in a controlled environment (within a docker container), I thought should be okay. But, if we feel this is mandatory, am fine with it.
But lets not remove these tests :-)